client/ios/Podfile.lock

```diff
@@ -1463,4 +1463,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: c02d6bf3ddbb12b31972cfec277c2a34a553e27f
 
-COCOAPODS: 1.12.0
+COCOAPODS: 1.11.3
```

Comment:
Would be great if you could update your cocoa pods :)

Reply:
Aha, didn't intend to commit this

Will update it too as you mentioned 👍